### PR TITLE
Add tests for LoadSampleDetailsFromRaw

### PR DESCRIPTION
**Pull request by gemmaguest**
This adds missing unit tests for the LoadSampleDetailsFromRaw algorithm.

**To test:**

Run LoadSampleDetailsFromRaw.

Refs #18215 .

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/DataHandling/test/LoadSampleDetailsFromRawTest.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#ifndef LOADSAMPLEDETAILSFROMRAWTEST_H_
+#define LOADSAMPLEDETAILSFROMRAWTEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidDataHandling/LoadSampleDetailsFromRaw.h"
+#include "MantidAPI/MatrixWorkspace.h"
```
Comment (PranavBahuguna) [inline Framework/DataHandling/test/LoadSampleDetailsFromRawTest.h]: You don't actually need this header, its already contained in the other ones.

**Comment by PranavBahuguna** [comment]
Tests look very good and concise, just see my one comment and its good to go.

File: Framework/DataHandling/test/LoadSampleDetailsFromRawTest.h (hunk 1 above)
Comment (gemmaguest) [inline Framework/DataHandling/test/LoadSampleDetailsFromRawTest.h]: Thanks - I've removed it.

**Comment by PranavBahuguna** [comment]
Alright all is good, :shipit: when checks pass.

**Pull request merged**